db/services.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+$functions = [
+
+        'mod_checkmark_get_checkmarks_by_courses' => [
+            'classname'     => 'mod_checkmark_external',
+            'methodname'    => 'get_checkmarks_by_courses',
+            'classpath'     => 'mod/checkmark/externallib.php',
+            'description'   => 'Get all checkmarks in the given courses',
+            'type'          => 'read',
+            'services' => array(MOODLE_OFFICIAL_MOBILE_SERVICE),
```

Comment:
Try to keep array definitions consistent, preferably  [] instead of array()
```suggestion
            'services' => [MOODLE_OFFICIAL_MOBILE_SERVICE],
```